extension-service-worker-explainer.md

```diff
@@ -0,0 +1,60 @@
+# Explainer for WebUSB & Extension Service Worker Integration
+
+A proposal to expose WebUSB in service workers created by extensions.
+
+<!-- TOC -->
+<!-- /TOC -->
+
+## Introduction
+The WebUSB API allows web pages to request permission to access a connected USB
+device and, once permission is granted, enables the web page to control the
+device. Extensions can currently access the API from the extension [background
+page](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/background).
+The goal of this proposal is to provide an alternate means for extensions to
+access WebUSB API without the use of a background page.
```

Comment:
Since Extensions can still create normal pages in tabs and popups this should be explicit that this is about access to the WebUSB API from the same kind of background context that was provided by the background page but is now a Service Worker.

Reply:
I've updated the introduction to explicitly mention that we are trying to provide access from a similar background context.